src/test/java/net/sf/jabref/importer/fileformat/IsiImporterTest.java

```diff
@@ -88,152 +134,175 @@ public void testProcessSubSup() {
+            Assert.assertEquals("2006", a.getField("year"));
+            Assert.assertEquals("#jul#", a.getField("month"));
+            Assert.assertEquals("89", a.getField("volume"));
+            Assert.assertEquals("4", a.getField("number"));
+
+            // JI Appl. Phys. Lett. (USA)
+
+            // BP 41103-1
+            // EP 41103-41103-3
+            // PS 41103-1-3
+            // assertEquals("41103-1-3", a.getField("pages"));
+
+            // LA English
+            Assert.assertEquals(
+                    "We report on the generation of continuous-wave (cw) ultraviolet"
+                            + " (UV) laser light at lambda =278 nm by optical frequency doubling of"
+                            + " visible light in beta -BaB$_2$O$_4$ waveguides. Ridge-type "
+                            + "waveguides were produced by He$^+$ implantation, photolithography"
+                            + " masking, and plasma etching. The final waveguides have core dimension"
+                            + " of a few mu m$^2$ and show transmission losses of 5 dB/cm at 532 nm "
+                            + "and less than 10 dB/cm at 266 nm. In our first experiments, a second "
+                            + "harmonic power of 24 mu W has been generated at 278 nm in an 8 mm long "
+                            + "waveguide pumped by 153 mW at 556 nm.".replaceFirst("266", "\n"),
+                    a.getField("abstract"));
+            /*
+             * DE Experimental/ barium compounds; ion implantation; optical harmonic
+             * generation; optical losses; optical pumping; photolithography; solid
+             * lasers; sputter etching; ultraviolet sources; waveguide lasers/
+             * second harmonic generation; continuous-wave light; beta -BaB/sub
+             * 2/O/sub 4/ optical waveguides; UV laser light; optical frequency
+             * doubling; visible light; ridge-type waveguides; He/sup +/
+             * implantation; photolithography masking; plasma etching; transmission
+             * losses; optical pumping; 278 nm; 532 nm; 266 nm; 24 muW; 8 mm; 153
+             * mW; 556 nm; BaB/sub 2/O/sub 4// A4265K Optical harmonic generation,
+             * frequency conversion, parametric oscillation and amplification A4255R
+             * Lasing action in other solids A4260B Design of specific laser systems
+             * B4340K Optical harmonic generation, frequency conversion, parametric
+             * oscillation and amplification B4320G Solid lasers/ wavelength
+             * 2.78E-07 m; wavelength 5.32E-07 m; wavelength 2.66E-07 m; power
+             * 2.4E-05 W; size 8.0E-03 m; power 1.53E-01 W; wavelength 5.56E-07 m/
+             * BaB2O4/ss B2/ss Ba/ss O4/ss B/ss O/ss C1 Degl'Innocenti, R.; Guarino,
+             * A.; Poberaj, G.; Gunter, P.; Nonlinear Opt. Lab., Inst. of Quantum
+             * Electron., Zurich, Switzerland
+             */
```

Comment:
Why is this long comment in the code? There are also other comments there. Why are they necessary? Or can they be removed?